server/routes/index.js

```diff
@@ -4,4 +4,11 @@
 
 'use strict';
 
-module.exports = [].concat(require('./base'), require('./auth'), require('./search'), require('./visits'));
+// TODO just sniff out all the files under /routes?
```

Comment:
thanks for the suggestions! `node-require-directory` shows me generated coffeescript, and `node-require` seems like massive overkill. I suspect there's a one-liner lurking here that'd be sufficient for our current case; care to take a swing?